packages/lib/services/rest/routes/notes.test.ts

```diff
@@ -0,0 +1,82 @@
+import Logger from '@joplin/utils/Logger';
+import shim from '../../../shim';
+import uuid from '../../../uuid';
+import { downloadMediaFile } from './notes';
+
+describe('services_rest_routes_notes', () => {
```

Comment:
For new tests, we should use "/" as separator, same as the path -  and just `routes/notes` is unique enough.

It means that on test failure, we can copy and paste the description that's displayed and run just that test. While if we copy and paste "services_rest_routes_notes" and run that it won't do anything, because it's looking up by file path.